### DISCUSSION TITLE
OAuth2 Token across Docker containers gives 401 Unauthorized

### DISCUSSION THREAD

**Discussion by Nihal987** [keycloak/keycloak#22400] [Q&A]
I have a keycloak clustered setup and I discovered that I cannot use the auth token from one instances for REST GET calls on the other instance. My three keycloak instances are running on docker containers on the same network. 

Does keycloak not support this under the default PING discovery protocol ?

**Comment by codespearhead** [comment]
It seems that your [the session cache is not available across the nodes](https://www.keycloak.org/server/caching).
